From 352df97848837ef59eccefde5883fda3696db174 Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Sat, 26 Jun 2010 00:25:42 +0800
Subject: [PATCH] ENGR00124683-2 Change adv7180_pwdn callback function on mx35

Change adv7180_pwdn callback function on mx35 due to pwdn parameter
defintion update.

Signed-off-by: Sammy He <r62914@freescale.com>
---
 arch/arm/mach-mx35/mx35_3stack.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx35/mx35_3stack.c b/arch/arm/mach-mx35/mx35_3stack.c
index 658bfb7..3208230 100644
--- a/arch/arm/mach-mx35/mx35_3stack.c
+++ b/arch/arm/mach-mx35/mx35_3stack.c
@@ -418,7 +418,7 @@ static struct mxc_fm_platform_data si4702_data = {
 
 static void adv7180_pwdn(int pwdn)
 {
-	pmic_gpio_set_bit_val(MCU_GPIO_REG_GPIO_CONTROL_1, 1, pwdn);
+	pmic_gpio_set_bit_val(MCU_GPIO_REG_GPIO_CONTROL_1, 1, ~pwdn);
 }
 
 static void adv7180_reset(void)
-- 
1.5.4.4

